.env.sample

```diff
@@ -15,6 +15,10 @@
 # (required) The domain of your canvas instance (e.g. canvas.instructure.com)
 # CANVAS_OAUTH_CANVAS_DOMAIN=
 
+# (optional) The Canvas API scopes needed by the application
```

Comment:
the CANVAS_OAUTH_SCOPES is "A list of Canvas API scopes that the access token will provide access to.", as specified in the [django-canvas-oauth library](https://github.com/Harvard-University-iCommons/django-canvas-oauth/blob/master/canvas_oauth/settings.py). 

So the scopes are associated with user OAuth tokens, not the API key as described in the #202?

Reply:
Well, it's both I think. Scopes on the API Developer Key in Canvas and in the application configuration need to match (if you are enforcing scopes). When the user authorizes the application, the OAuth process checks whether the scopes match, and then mints an access token and refresh token pair which are associated with the scopes at the time they were created. The access token can only be used to make requests to those endpoints/scopes.

Reply:
Like @jonespm said, I think it's possible the application requests fewer scopes than are enabled on the Canvas API Developer Key side, and it should still work. I'll test it to be sure.

Reply:
Yeah, it seems to work if you add a new scope that you don't end up using; OAuth process and requests work. I assume if you tried to use that missing scope using Postman or some application process, you'd still get an "insufficient scopes on token" error.

Reply:
Got it!